app/assets/v2/css/card.css

```diff
@@ -1,9 +1,10 @@
 .toggle {
-  margin-top: 10px;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  font-weight: 600;
+  display: inline-flex;
```

Comment:
can we create an extra class instead of overwrite?

Reply:
or if our component gc-toggle but personally I will say the best will be using https://getbootstrap.com/docs/4.5/components/forms/#switches

Reply:
donezo